### DISCUSSION TITLE
Blick nach hinten im Fahrzeug

### DISCUSSION THREAD

**Discussion by micknessi** [Mogli12/KeyboardSteer#791] [General]
Gibt es eine VCA-Version vor der 2.0 für den LS22, bei der es schon die Parkbremse gibt?
bei der 2.0 geht der Blick nach hinten im Fahrzeug beim Rückwärtsfahren in Verbindung mit Head-Tracking nicht.

**Comment by Mogli12** [comment]
In Verbindung mit Head-Tracking ändert VCA die Kamera nicht. Das gibt zu viele Probleme.

**Reply by micknessi** [reply]
Danke für die Info.
Ich benutze zur Zeit Version 1.0 RC1, da funktioniert das. Leider gibt es da aber eben noch nicht die Parkbremse. Wäre halt schön wenn beides ginge

**Comment by Mogli12** [comment]
Ich habe das als Reaktion auf https://github.com/Mogli12/VehicleControlAddon/issues/714 ausgebaut.

**Reply by micknessi** [reply]
Habe ich gelesen, das hatte ich zwar auch, war mir aber egal. Trotzdem schade.
Bitte nicht aufhören oder entmutigen lassen, tolle Arbeit bisher!!

**Comment by Mogli12** [comment]
Mit Version 102 ist es wieder eingebaut. Bitte schaue ggf. in die globalen Einstellungen von VCA.
